libtransmission/makemeta.h

```diff
@@ -180,43 +182,19 @@ class tr_metainfo_builder
 
     ///
 
-    [[nodiscard]] constexpr static uint32_t default_piece_size(uint64_t total_size) noexcept
+    [[nodiscard]] inline static uint32_t default_piece_size(uint64_t total_size) noexcept
```

Comment:
_(nit)_  I'd lean towards moving this to the .cc file so that we don't have to add `<cmath>` and `<tr-assert>` into the public header. This function is almost never called so there's no performance benefit to inlining it.

Reply:
Sure thing.